tests/sorcha/test_PPApplyFOVFilter.py

```diff
@@ -92,7 +92,7 @@ def test_PPApplyFOVFilters():
     configs = {
         "camera_model": "footprint",
         "footprint_path": get_test_filepath("detectors_corners.csv"),
-        "footprint_edge_threshold": 0.0,
+        "footprint_edge_threshold": 10.0,
```

Comment:
Why is the footprint edge on here? There is a test that checks this I thought separately?

Reply:
To make sure that this test actually does something. I noticed while I was running this test that the tested objects _all_ make it through the footprint filter. Setting the edge threshold to high at least removes a couple of them and ensures that the footprint filter is activating properly with the right config keys, which is all that is being tested here.

Reply:
But testing the foorprint edge is a different test, right? Shouldn't this be testing the generic part of the footprint fitler?

Reply:
I have brought the tests in PPApplyFOVFilter in line with the tests on its child functions: so now they all test exactly the same data and give exactly the same results. The part that tests "footprint" was copied directly from Sam's test_PPApplyFootprint.py.

Turns out I wrote the tests for PPSimpleSensorArea and PPCircleFootprint, so I double-checked them while I was there. All good and as expected. I've just standardised them so all tests use the first ten rows of the same data file.